website/templates/index.mako

```diff
@@ -2,122 +2,326 @@
+
+    <div class="container">
+      <div class="feature-2 space-top space-bottom">
+        <div class="row">
+          <div class="col-xs-12 text-center headline">
+            <h2>Integrations make your <strong>workflow more efficient</strong></h2>
+          </div>
         </div>
+        <div class="row integrations">
+          <div class="col-sm-3 col-xs-6">
+            <img src="/static/img/front-page/dropbox.png" class="img-responsive"/>
+          </div>
+          <div class="col-sm-3 col-xs-6">
+            <img src="/static/img/front-page/github.png" class="img-responsive"/>
+          </div>
+          <div class="col-sm-3 col-xs-6">
+            <img src="/static/img/front-page/amazon.png" class="img-responsive"/>
+          </div>
+          <div class="col-sm-3 col-xs-6">
+            <img src="/static/img/front-page/box.png" class="img-responsive"/>
+          </div>
+      </div>
+      <div class="row integrations">
+          <div class="col-sm-3 col-xs-6">
+           <img src="/static/img/front-page/google.png" class="img-responsive"/>
+          </div>
+          <div class="col-sm-3 col-xs-6">
+            <img src="/static/img/front-page/figshare.png" class="img-responsive"/>
+          </div>
+          <div class="col-sm-3 col-xs-6">
+            <img src="/static/img/front-page/dataverse.png" class="img-responsive"/>
+          </div>
+          <div class="col-sm-3 col-xs-6">
+            <img src="/static/img/front-page/mendeley.png" class="img-responsive"/>
+          </div>
+      </div>
+      <div class="row">
+          <div class="col-xs-12 text-center feature-2">
+            <p>Open Science Framework integrates with your existing work platforms making collaboration easier.</p>
+          </div>
+      </div>
+      </div>
     </div>
-    <div class="row text-center">
-        <div class="col-md-12">
-            <p class="hpSubHeadThree hpSteps"><a class="hpLink" href="#signUp">Sign up.</a></p>
-            <p class="hpSubHeadThree hpSteps"><a class="hpLink" href="/getting-started/">Learn how to build a project.</a></p>
-            <p class="hpSubHeadThree hpSteps"><a class="hpLink" href="/explore/activity/">Get inspired.</a></p>
+
+    <div class="feature-3">
+      <div class="container">
+        <div class="row space-top">
+
+          <div class="col-md-12 text-center space-bottom">
+            <h2><strong>Everything</strong> your research needs to be a success</h2>
+          </div>
         </div>
+        <div class="row space-bottom">
+          <div class="col-sm-6 text-right">
+            <h3>Manage your project</h3>
+            <p>View all of your projects from <strong>one dashboard.</strong></p>
+
+            <h3>Quickly share files</h3>
+            <p><strong>Share key project information</strong> and allow others to use and cite it.</p>
```

Comment:
This text doesn't seem to match the heading; sharing files is more about sharing data than sharing project information.